Makefile

```diff
@@ -41,7 +41,7 @@ export CTR_CMD     ?= $(or $(shell command -v podman), $(shell command -v docker
 # E.g. --tls-verify=false for local develop when using podman
 CTR_CMD_PUSH_OPTIONS ?=
 
-GENERAL_TAGS := include_gcs include_oss containers_image_openpgp gssapi providerless netgo osusergo
+GENERAL_TAGS := include_gcs include_oss containers_image_openpgp gssapi providerless netgo osusergo bpf
```

Comment:
nooo! we've only just escaped the build tag

Reply:
yeah - I know. 
I guess I can't see a way around this without build tags, but I do think this is a valid use case for the build tag if we don't want to check in generated objects into the codebase